### DISCUSSION TITLE
I am getting ERROR: No enum constant org.keycloak.theme.Theme.Type.ADMİN error.

### DISCUSSION THREAD

**Discussion by Mert-Filiz** [keycloak/keycloak#24827] [Archive]
Hi,

I am new in the keycloak world and am beginning my learning process. However, I have some issues when I start the keycloak server on my local machine. When I try to run the ./kc.sh start-dev command I am getting this error shown below.

ERROR: Unexpected error when starting the server in (development) mode
ERROR: Failed to start quarkus
ERROR: Failed to load themes
ERROR: No enum constant org.keycloak.theme.Theme.Type.ADMİN

I am working on macOS Ventura Version 13.5.2 And I am using keycloack version 22.0.5.

Does anyone know the solution?

Thanks.

**Comment by ElifSena-cal** [comment]
Hello, I also encountered the same problem. When I changed the computer language to English, it worked. Maybe you have the same issue.

**Comment by Mert-Filiz** [comment]
It worked. Thank you for your help !!!

**Reply by ElifSena-cal** [reply]
You're welcome
